src/components/ScholarshipList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './scholarshipList.css';
+
+function ScholarshipList(props){
+    return(
+        <div class="scholarship-row">
+            <ul>
+                <li>{props.name}</li>
+                <li>{props.amount}</li>
+                <li>{props.deadline}</li>
+                <li>favorte btn</li>
+            </ul>
+        </div>
+    );
```

Comment:
Maybe instead of taking in individual scholarships, this component can take in a list of scholarships and return the entire list? I'm looking at the [Basic List Component](https://reactjs.org/docs/lists-and-keys.html#basic-list-component) section from the [List and Keys](https://reactjs.org/docs/lists-and-keys.html) concepts page. 

I think also we might also need [Keys](https://reactjs.org/docs/lists-and-keys.html) on each of the items.